src/main/java/no/neic/tryggve/constants/ConfigName.java

```diff
@@ -3,7 +3,6 @@
 public final class ConfigName {
     public static final String HOST = "host";
     public static final String HTTP_VERTICLE_PORT = "http_verticle_port";
-    public static final String SSL = "ssl";
```

Comment:
Okey. Its confused me since it did not have anything to do with the documentation. This commit should not have been part of this pull req.